lib/soapy_cake/admin.rb

```diff
@@ -1,4 +1,5 @@
 module SoapyCake
+  # rubocop:disable Metrics/ClassLength
```

Comment:
I really hate doing this, there are actually a few methods here which could be refactored into separate classes (like accounting/signup methods). But I really don't want to introduce breaking API changes in this bug fix...